.github/workflows/apidiff.yaml

```diff
@@ -31,3 +31,14 @@ jobs:
         name: pkg.main
     - name: Compare regenerated code to baseline
       run: apidiff -incompatible pkg.main ./... > diff.txt && cat diff.txt && ! [ -s diff.txt ]
+    - name: Add breaking change label
+      if: ${{ failure() && !github.event.pull_request.head.repo.fork }}
+      uses: actions/github-script@v6
+      with:
+        script: |
+          github.rest.issues.addLabels({
+            issue_number: ${{ github.event.number }},
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            labels: ['contains breaking change']
```

Comment:
nit: for brevity maybe just "breaking change". Contains seems implied by adding the label.